clouway-pcache-client-gae/src/main/java/com/clouway/api/pcache/extensions/gae/GAECacheManager.java

```diff
@@ -72,7 +75,21 @@ public Object get(String key) {
     return result;
   }
 
-  /**
+    @Override
+    public List<Object> getAll(List<String> keys) {
```

Comment:
I think that we can make it a little bit better, we can return an Hits(HitResult) that could return and the keys that was not found
```kotlin
val hits = cache.getAll(keys)

if(hits.missed.isEmpty) {
 return hits.addAll(origin.get(hits.missed)).toList() 
} else {
 return hits.toList()
}
```
the names are not the best

other suggestion could be 
```kotlin
val hits = cache.getAll(keys, {missedKeys ->  origin.get(missedKeys) })
```
what do you think? 

maybe we can have 2 methods